### ISSUE TITLE
[Bug]: Pie chart over lapping

### ISSUE THREAD

**Issue by nbukhari55** [jasp-stats/jasp-issues#3081]
### JASP Version

0.19.1

### JASP Module

Descriptives

### What OS are you seeing the problem on?

macOS Silicon

### Bug Description

The pie charts are overlapping
![jasp test](https://github.com/user-attachments/assets/638b38f4-6e44-4cb7-93b1-a6633badd6fb)

### Expected Behaviour

it should not overlap

### Steps to Reproduce

I enter the data to do the analysis again and the pie charts overlap again

### Final Checklist

- [X] I have included a screenshot showcasing the issue, if possible.
- [X] I have included a JASP file (zipped) or data file that causes the crash/bug, if applicable.
- [X] I have accurately described the bug, and steps to reproduce it.

**Comment by tomtomme** [comment]
@nbukhari55 
Yes this is annoying.
As a workaround you have two options atm:
1. Shorten the question to one or 2 words
2. export the pie chart to pptx format and edit it in Powerpoint.

I cannot give a timeline on when this might be fixed because we have many higher priority bugs that cannot be worked around by the user.

**Comment by nbukhari55** [comment]
yes, it is very annoying - for now, I will just skip using the pie chart feature and use bar charts, etc.

**Comment by tomtomme** [comment]
just as a quick update - with latest module beta the pie chart has this new problem:

<img width="943" height="915" alt="Image" src="https://github.com/user-attachments/assets/f31adef1-f581-47cd-82ec-3d8f9abf897c" />

need to re-check with 0.97 and dynamic plots

**Comment by tomtomme** [comment]
closing in favor of https://github.com/jasp-stats/jasp-issues/issues/4287
